### PR TITLE
Feat: Elements Early Access

### PR DESCRIPTION
**Pull request by paulatulis**
Setting up early access landing page for Elements.

### CODE REVIEW

**Comment by lottamus** [comment]
Deploy preview for *stoplightio* ready!

Built with commit d68b9c266f17eb8c2723039fed51da2730585064

https://deploy-preview-1660--stoplightio.netlify.app

File: netlify/pages/elements.yaml [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+      title: Get the API Design Guide
+      type: submit
+bodyImage:
+  alt: Elements
+  src: /images/elements/elements-new.png
+subFeatures:
+  title: Embed API Docs Anywhere
+  description: >-
+    Whether you have an existing developer portal or are working locally, you can use these components to add an
+    automatically generated API reference that fits seamlessly.
```
Comment (philsturgeon) [inline netlify/pages/elements.yaml:32]:
```suggestion
    Whether you have an existing developer portal, or are working locally, you can use these components to add
    automatically generated API reference documentation that fits in seamlessly.
```

File: netlify/pages/elements.yaml [hunk 2]
```diff
@@ -0,0 +1,153 @@
...
+  features:
+    - category:
+      title: Best-in-class OpenAPI Support
```
Comment (philsturgeon) [inline netlify/pages/elements.yaml:35]:
```suggestion
      title: Powered by Standards
```

File: netlify/pages/elements.yaml [hunk 3]
```diff
@@ -0,0 +1,153 @@
...
+      description: >-
+        Generate comprehensive API References from OpenAPI documents with support for advanced features like alloff,
+        ref, discriminators and much more.
```
Comment (philsturgeon) [inline netlify/pages/elements.yaml:38]:
```suggestion
        Leverage OpenAPI and JSON Schema to create beautiful, templated documents for even the most dynamic payloads, end encourage reuse with $ref.
```

Comment (philsturgeon) [reply netlify/pages/elements.yaml:38]: We actually dont support discriminator anywhere which is why I mentioned this 😅

File: netlify/pages/elements.yaml [hunk 4]
```diff
@@ -0,0 +1,153 @@
...
+    - category:
+      title: Code Samples
+      description: >-
+        Do the heavy lifting for developers and kickstart integration by providing code samples in popular languages
+        like Curl, Python, Ruby, Java, and many more.
+      image: /images/elements/code.svg
+    - category:
+      title: Enhanced Markdown Support
+      description: >-
+        Write rich tutotials and descriptions with Stoplight-flavored markdown that gives the ability to add tabs, callouts, embed
```
Comment (philsturgeon) [inline netlify/pages/elements.yaml:54]:
```suggestion
        Write rich tutotials and descriptions with Stoplight-flavored Markdown that gives the ability to add tabs, callouts, embed
```

Usual advice is to try and use capitals for proper nouns, like Markdown, JSON Schema, etc. The sort of nerds who care about standards care about that sort of thing _a lot_. :)

File: netlify/pages/elements.yaml [hunk 5]
```diff
@@ -0,0 +1,153 @@
...
+        JSON examples, and much more.
+      image: /images/elements/markdown-mark.svg
+    - category:
+      title: Easy to Set Up
+      description: >-
+        No build steps. Get started in minutes. Built on web components, it's as easy as adding a tag within your
+        project. It can be used with any framework.
```
Comment (philsturgeon) [inline netlify/pages/elements.yaml:61]:
```suggestion
        project, and can be used with any framework.
```

File: netlify/pages/elements.yaml [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+      image: /images/elements/stopwatch.svg
+    - category:
+      title: Delightful DX
+      description: >-
+        Help developers find and consume APIs faster with a customizable left navigation and three-column layout.
+      image: /images/elements/smile-beam.svg
+integrations:
+  title: Not using other Stoplight tooling? 
+  summary: >-
+    Work directly with OpenAPI files or use Stoplight to manage OpenAPI files and behavioral docs. The choice is yours.
```
Comment (philsturgeon) [inline netlify/pages/elements.yaml:71]:
```suggestion
    Use any text-editor, IDE or OpenAPI editor you like, not just Stoplight Studio. There's no vendor lock-in required,
    just convenient optional integration. Point Elements at your OpenAPI and Markdown files. 
```

**Review by philsturgeon — APPROVED** [review]

File: src/components/Header/Desktop.tsx [hunk 7]
```diff
@@ -165,6 +165,16 @@ const OpenSourceLinks = () => {
...
         />
+
+        <ProductLink
+          name="Elements"
+          description="Web components for customizable and interactive API Documentation ."
+          className="ml-4"
+          color="green"
+          href="/open-source/elements"
+          icon={['fad', 'paint-brush']}
+          tag="Elements"
```
Comment (philsturgeon) [inline src/components/Header/Desktop.tsx:176]:
```suggestion
          tag="Documentation"
```

Is it this which changes this?

**Comment by philsturgeon** [comment]
One small tweak but lets push to get this in this week. I'd love to start talking about this publicly!

File: netlify/pages/elements.yaml [hunk 8]
```diff
@@ -0,0 +1,153 @@
...
+    - title: Maintained by Stoplight
+    - title: Free Forever
+  ctas:
+    - submit:
+        button:
+          title: Get Early Access
+          color: blue2
+        formId: 22bf0180-f3dd-4113-ac29-f68a7f408cb4
+        input:
+          placeholder: Your work email...
```
Comment (philsturgeon) [inline netlify/pages/elements.yaml:21]:
```suggestion
          placeholder: Your email...
```

In the footer signup we just say "Your email" so this makes it consistent.

**Comment by philsturgeon** [comment]
@paulatulis thanks for this great work, I'm going to merge it because I am excited to get this in front of people's faces!

**Comment by paulatulis** [comment]
> @paulatulis thanks for this great work, I'm going to merge it because I am excited to get this in front of people's faces!

Hey @philsturgeon sounds good! I may need to swap out the form because that one is a placeholder, but can take care of it after this merges.

**Comment by philsturgeon** [comment]
@paulatulis oh! Might be better to get that done first, as that form is the main use for this page right now. Do you think you could get it sorted today or Monday?

**Comment by paulatulis** [comment]
> @paulatulis oh! Might be better to get that done first, as that form is the main use for this page right now. Do you think you could get it sorted today or Monday?

I can definitely get it done now, but need the preview to load first. I'm also going to remove the auto merge.

**Comment by paulatulis** [comment]
@philsturgeon actually this is something I can work on in HubSpot separately. It's good to go!

**Comment by philsturgeon** [comment]
Thank you!

**Pull request merged**